javascripts/govuk/anchor-buttons.js

```diff
@@ -0,0 +1,55 @@
+(function(global, customConfig) {
```

Comment:
Can we add a comment block at the top explaining what the file does? E.g. https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/stop-scrolling-at-footer.js